src/apps/service/apps/jobs.clj

```diff
@@ -181,10 +173,7 @@
 
 (defn submit
   [apps-client user submission]
-  (transaction
```

Comment:
This transaction had to be removed from here, since async sub-job submissions cannot share the same transaction as the parent call that responds to the client immediately, because the parent's db connection will close after the response is sent to the client.